### ISSUE TITLE
Setting min and max duration of audiotracks not working as expected

### ISSUE THREAD

**Issue by gavspav** [roundware/roundware-web-template#24]
As ever, everything which follows has a huge caveat of it probably being user error......

If I set the min and max playback duration to 10 seconds I expect (after reading the docs) clips < 10s to play in their entirety and longer clips to just play for 10 seconds.

However what actually happens is that none of the clips play.

**Comment by hburgund** [comment]
Your expectation is correct! Not sure why this isn't working properly; could be a bug or could be that your fade times are too long. If your fade in/out times are say 10s each, then each clip is going to play, start fading in a bit and then fade out again before it is even audible.

What value of playback duration do things start working again?

**Comment by hburgund** [comment]
OK, agreed that this seems like a bug then. Probably just some simple math wrong in the playback pipeline that only becomes obvious at the shorter lengths.

If there are any additional consistencies you can discover, please note them here as that will help us figure it out.

To be clear, are you saying that when the min time is set to 0 and max is 10, things work OK? It's just when they are both set to 10 that this issue happens?

@shreyas-jadhav we can check this out when you have the time.

**Comment by gavspav** [comment]
Yes I think it's just when you set min and max to the same. The markers behave as expected ie stay red for n seconds. But there is no audio,
